tests/tests.cpp

```diff
@@ -22,6 +22,34 @@ TEST(tests, issueBar){
 }
 extern "C" int craft_main(int argc, char* argv[]);
 
+///[issue]https://github.com/Team-10-But-Better/Craft/issues/11
+TEST(tests, Issue11){
+	#ifdef __linux__
+	{
+		EXPECT_TRUE(true);
+		EXPECT_EQ(system("mpg321 -h > /dev/null 2>&1"),0) << "This indicates you have failed to install mpg321 package as instructed for audio. Please install this dependency";	
```

Comment:
I am not sure we should be requiring the users to install new packages, especially since you do not change the README to indicate that they need to install mpg321.

Reply:
I did indicate in the README that the user needs to install mpg321, the README that is in the Craft directory & will be the first README the user sees when accessing the project on Git-Hub without delving into the folders. Also mpg321 is crucial in order for the audio to work. This will not be changed.